pkg/controllers/reallocation/utilization.go

```diff
@@ -68,7 +68,7 @@ func (u *Utilization) markUnderutilized(ctx context.Context, provisioner *v1alph
 		if err := u.kubeClient.Patch(ctx, node, client.MergeFrom(persisted)); err != nil {
 			return fmt.Errorf("patching node %s, %w", node.Name, err)
 		}
-		zap.S().Debugf("Added TTL and label to underutilized node %s", node.Name)
+		zap.S().Infof("Added TTL and label to underutilized node %s", node.Name)
```

Comment:
I think the two log lines in this file should be Debug. I think that adding TTL and removing TTL could happen a lot, and don't think it's as important as creating and terminating an instance.

Reply:
Happy to discuss offline. I'm not sure this should be happening a lot, and if it is, I think it makes sense to be visible.